packages/shared/lib/services/connection.service.ts

```diff
@@ -477,6 +477,10 @@ class ConnectionService {
             .update({ metadata });
     }
 
+    public async bulkReplaceMetadata(ids: number[], metadata: Metadata) {
```

Comment:
can we refactor `replaceMetadata` to accept a array of connections?

Reply:
Sure: https://github.com/NangoHQ/nango/pull/2145/commits/294aa89bed2aa5984e8072081dbb096eb986040e